.gitmodules

```diff
@@ -21,4 +21,4 @@
 	branch = v0.22.0
 [submodule "tests/jsondata"]
```

Comment:
testdata

Reply:
We would better to finish this change in new pr,  since `vm_test` will influenced.